wiki/bin/standupbot.py

```diff
@@ -0,0 +1,155 @@
+# StandupBot
+
+import re
+import json
+import urllib2
+import random
+import datetime
+
+from mediawiki import MediaWiki, CONFIG
+
+# Handy flags to help with debugging...
+# Don't actually push messages to Slack, just show the message payload locally
+DO_NOT_SEND = False
+# Add all standups to the announce list, regardless of time
+ANNOUNCE_ALL = False
+# Announce in a specific channel, rather than the one listed in the Wiki
+# False is a default value, but '@yourusername' and '#sandbox' are also good
+ANNOUNCE_IN = False
+
+print 'StandupBot'
```

Comment:
print() is a function (so on py2 you'll need to `from __future__ import print_function`).
There is a `2to3` script python comes with that gives you some hints, but doesn't maintain 2-compatibility. The only issues I can see are print(), and you'll need to handle the differing handling of urllib2/urllib.